config_defaults/subtests/docker_cli/start.ini

```diff
@@ -8,6 +8,8 @@ docker_run_timeout = 30.0
 docker_attach = yes
 #: Run interactive container -i
 docker_interactive = yes
+#: docker command wait time
```

Comment:
@cevich @edsantiago - I changed it to what Ed suggested and it's https://github.com/autotest/autotest-docker/pull/675